easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
```

Comment:
Well, we currently already don't use the block in all easyconfigs? And then I need to change the easyconfig and stuff. It's a lot of hassle.